application/classes/Ushahidi/DataProvider.php

```diff
@@ -43,7 +43,7 @@ public function receive($type, $from, $message, $to = NULL, $title = NULL, $data
 		$usecase = service('factory.usecase')->get('messages', 'receive');
 		try
 		{
-			$usecase->setPayload(compact(['type', 'from', 'message', 'to', 'title', 'data_provider_message_id', 'data_provider', 'contact_type']))
+			$message_id = $usecase->setPayload(compact(['type', 'from', 'message', 'to', 'title', 'data_provider_message_id', 'data_provider', 'contact_type']))
```

Comment:
Do we need this change? if so can we update the docblock to with a correct @return